RenderStateNotationParser/src/RenderStateNotationParserImpl.cpp

```diff
@@ -159,25 +158,70 @@ static void Deserialize(const nlohmann::json& Json, RayTracingPipelineNotation&
         Deserialize(Json["MaxAttributeSize"], Type.MaxPayloadSize, Allocator);
 }
 
+bool IncludeHandlerRenderStateNotationParser::InsertPath(std::string const& Path)
+{
+    auto Iter = m_IncludePaths.insert(Path);
+    return Iter.second;
+}
+
+bool IncludeHandlerRenderStateNotationParser::IsContain(std::string const& Path) const
+{
+    auto Iter = m_IncludePaths.find(Path);
+    return Iter != m_IncludePaths.end() ? true : false;
+}
```

Comment:
I don't think wrapping these functions adds value.
Using unordered_set directly will be clearer.